beacon_node/lighthouse_network/Cargo.toml

```diff
@@ -42,14 +42,23 @@ superstruct = { workspace = true }
 prometheus-client = "0.22.0"
 unused_port = { workspace = true }
 delay_map = { workspace = true }
-void = "1"
-libp2p-mplex = { git = "https://github.com/sigp/rust-libp2p/", rev = "b96b90894faab0a1eed78e1c82c6452138a3538a" }
-
-[dependencies.libp2p]
-git = "https://github.com/sigp/rust-libp2p/"
```

Comment:
we can depend on `sigp/rust-libp2p` `master` after https://github.com/sigp/rust-libp2p/pull/566 gets merged